.github/workflows/prepare_release.yml

```diff
@@ -32,6 +32,11 @@ jobs:
         run: |
           cd dev_tools/ && bin/bump-docker-image-version \
             ${{ steps.pipeline_context.outputs.candidate_version }}
+      - name: "Bump installer starter frontend version"
+        if: github.ref_name == 'master'
+        run: |
+          cd dev_tools/ && bin/bump-starter-frontend-version \
+            ${{ steps.pipeline_context.outputs.candidate_version }}
```

Comment:
I think we should use [`candidate_patch_branch`](https://github.com/solidusio/solidus/blob/04a4d68ef9c6957a88e6d66618397754e58367c0/dev_tools/lib/solidus/pipeline_context.rb#L72). `candidate_version` would return `3.4.0`, while `candidate_patch_branch` would return `v3.4`.

Reply:
You are rigth!

Reply:
Fixed!